services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeServiceImpl.kt

```diff
@@ -78,11 +90,15 @@ class HankeServiceImpl (@Autowired val hankeRepository: HankeRepository) : Hanke
         logger.info {
             "Saving  Hanke step 2 for: ${entity.hankeTunnus}"
         }
+
```

Comment:
Add a comment here, about needing a transaction around these (or the whole request-reponse chain). Maybe the framework is already doing it for us (or can do it if configured so), but we haven't checked that yet.